kolibri_explore_plugin/assets/src/views/SearchPage.vue

```diff
@@ -59,18 +67,7 @@
       </div>
     </div>
 
-    <div v-if="recommended" class="flex-shrink-0 mt-auto recommended">
-      <b-container v-if="recommended" class="pb-5 pt-3">
-        <h4 class="text-muted">
-          Featured Channels
-        </h4>
-        <ChannelCardGroup
-          :rows="recommended"
-          :columns="columns"
-          @card-click="goToChannel"
-        />
-      </b-container>
-    </div>
```

Comment:
I see recommended channels in the specs, even for the minimal version:
![Captura de pantalla de 2021-07-12 10-55-21](https://user-images.githubusercontent.com/83944/125299668-bd930980-e2ff-11eb-82cc-209d6ae7e3bd.png)

So I would keep this.

Reply:
This is for the "No results", that's a bit different from the "Empty state". In any case. I've recovered the recommended in the code and just show it when the result is empty, but not in the first view with an empty search query.